lib/AACStream/metadata-stream.js

```diff
@@ -0,0 +1,229 @@
+/**
```

Comment:
This is a slightly altered metadata-stream since it looks for the timestamp ID3 tags and then sends them backward through the pipeline. I can however alter the file in the m2ts folder, but then I believe the file would be in an incorrect folder since it is being used by files outside the m2ts directory.